src/poprox_recommender/handler.py

```diff
@@ -60,3 +66,49 @@ def generate_recs(event, context):
     response = {"statusCode": 200, "body": json.dumps(body)}
 
     return response
+
+def classify_news_topic(model, tokenizer, general_topics, topic):
```

Comment:
Something I'm confused about reading this code is how the model knows which label corresponds to which general topic. I see that the number of labels to classify with matches the number of general topics but I don't see where the model gets any general topic information it can use to figure out which labels are relevant.

Reply:
Ahh I see your point! Yep after reading it I do feel the general_topics is missing from the classificatino step, I just updated the logic along with a pre-fine-tuned language model based on news categories and now the test result looks much better. I'll commit the changes later.